server/aws-lsp-codewhisperer/src/language-server/chat/chatController.test.ts

```diff
@@ -95,6 +103,18 @@ describe('ChatController', () => {
             testFeatures.credentialsProvider
         )
 
+        const mockCredentialsProvider: CredentialsProvider = {
+            hasCredentials: sinon.stub().returns(true),
+            getCredentials: sinon.stub().returns({ token: 'token' }),
+            getConnectionMetadata: sinon.stub().returns({
+                sso: {
+                    startUrl: undefined,
+                },
+            }),
+        }
+
+        telemetryService = new TelemetryService(mockCredentialsProvider, 'bearer', {} as Telemetry, {})
+        invokeSendTelemetryEventStub = sinon.stub(telemetryService, 'sendTelemetryEvent' as any)
```

Comment:
Did you plann to add unit tests for new TelemetryService invocation here? Let's add it as part of the change introducing new call

Reply:
not yet but can add those later and create a task,whereever telemetry service is being invoked.